packages/Frontend/src/containers/AuthenticateContainer.test.jsx

```diff
@@ -1,26 +1,15 @@
 /* eslint-env jest */
 import React from 'react';
 import { shallow } from 'enzyme';
-import { Auth } from 'aws-amplify';
+import { auth } from 'firebase';
 import { createAssertWithPropsToMatchSnapshot } from '../../test-utils';
 import AuthenticateContainer from './AuthenticateContainer';
 
-jest.mock('aws-amplify', () => ({
-  Auth: {
-    currentAuthenticatedUser: jest.fn(() => Promise.resolve({ username: 'user' })),
-    signIn: jest.fn((username, password) => new Promise((resolve, reject) => {
-      if (username === 'user' && password === 'password') {
-        resolve();
-      }
-      reject();
-    })),
-  },
-}));
+jest.mock('firebase');
```

Comment:
This tells jest we want it to not actually use firebase, we want to use the mock I made in another file.